docs/manage/data/dataset.md

```diff
@@ -67,6 +73,51 @@ Repeat this with all images in your dataset.
 
 To delete a tag, click the cross next to the corresponding label.
 
+## Datasets
+
+{{< alert title="Support" color="info" >}}
+Datasets currently only work for images.
+{{< /alert >}}
+
+### Create a dataset
+
+To create a dataset, go to the [**DATA** tab](https://app.viam.com/data/view) in the Viam app.
+
+On the **DATASETS** subtab, click **Create dataset**, enter a name for your dataset and click **Create dataset** again.
+
+![Create dataset UI](/manage/data/create-dataset.png)
+
+you can filter available images, using the **Filtering** menu and select the attributes that match where, how, and when the data was collected.
+
+### Add or remove data from a dataset
+
+To add or remove data from a dataset, go to the [**DATA** tab](https://app.viam.com/data/view) in the Viam app.
+
+On the **Images** subtab, you can filter available images, using the **Filtering** menu and select the attributes that match where, how, and when the data was collected.
+
+To add image to a dataset, click on the image and select the dataset in the Datasets dropdown.
+You can select an existing dataset or create a new one from this dropdown.
+
+{{<gif webm_src="/manage/data/add-to-dataset.webm" mp4_src="/manage/data/add-to-dataset.mp4" alt="Add image to dataset">}}
+
+To remove an image from a dataset click on the delete button next to the dataset.
+
+{{<gif webm_src="/manage/data/delete-from-dataset.webm" mp4_src="/manage/data/delete-from-dataset.mp4" alt="Remove from dataset">}}
+
+### View the data in a dataset
+
+Once you have added data to your dataset, you can view the data from the **DATASETS** tab by clicking on your dataset.
+
+You can also add or remove labels or tags from this menu or remove data from the dataset by clicking on an image and using the appearing menu.
```

Comment:
This took me a few reads — perhaps:

`
You can click on any image in the dataset and use its appearing menu in order to add or remove bounding box labels or tags, or remove that image from the dataset
`

Reply:
sounds good thx!